domain/src/main/java/life/qbic/domain/usermanagement/UserActivated.java

```diff
@@ -0,0 +1,80 @@
+package life.qbic.domain.usermanagement;
+
+import java.io.Serial;
+import java.time.Instant;
+import life.qbic.domain.events.DomainEvent;
+
+/**
+ * <b>A user was activated.</b>
+ */
+public class UserActivated extends DomainEvent {
+
+  public enum Reason {
+    EMAIL_CONFIRMED
+  }
```

Comment:
This is information we should not expose to subscribers imo. The reason of the activation is not relevant, and thats not what the event is about.